tests/foreman/cli/test_report.py

```diff
@@ -52,8 +52,9 @@ def test_positive_CRD_satellite(run_puppet_agent, session_puppet_enabled_sat):
 
 @pytest.mark.e2e
 @pytest.mark.rhel_ver_match('[^6]')
+@pytest.mark.parametrize('puppet_agent', ['7', '8'], ids=['puppetagent7', 'puppetagent8'])
```

Comment:
Here I want to have more generic parametrization `'client_ver', ids=['client1','client2']` that would suite to ALL client tests not just puppet.

Reply:
For client RHEL version parametrization we have single concept.  
Why can't we have single parametrization for client version ?